part-spec/Registers.json

```diff
@@ -0,0 +1,88 @@
+{
+    "registerDefiniton": {
+        "type": "object",
+        "required": [
+            "registerName",
+            "registerAddressOffset",
+            "registerSize"
+        ],
+        "properties": {
+            "registerName": {
+                "description": "Name of a register",
+                "type": "string"
+            },
+            "registerLongName": {
+                "description": " Full Name of a register",
+                "type": "string"
+            },
+            "registerAddressOffset": {
+                "description": "Address of a register",
+                "type": "number",
+                "comment": "Hex numeral system"
+            },
+            "registerSize": {
+                "description": "register Size",
+                "type": "number",
+                "comment": "units of bytes"
+            },
+            "registerType": {
+                "description": "Type of a register",
+                "type": "string",
+                "comment": "examples include but are not limited to data registers, address registers"
+            },
+            "registerResetValue": {
+                "description": "Reset value of a register",
+                "type": "number",
+                "comment": "Hex numeral system"
+            },
+            "registerValue": {
```

Comment:
why would we have registerValue as a separate thing from the "reset value of the register". How would this get set? Are you thinking this is associated with some sort of FW/SW?

Reply:
Yes, this was an input from someone from FW/SW.